vector/src/main/java/im/vector/app/features/home/room/detail/timeline/url/PreviewUrlView.kt

```diff
@@ -76,6 +81,22 @@ class PreviewUrlView @JvmOverloads constructor(
         }
     }
 
+    override fun render(messageLayout: TimelineMessageLayout) {
```

Comment:
weird to have to `render` fun in this class now

Reply:
Changed to renderMessageLayout